tests/testthat/_snaps/object.md

```diff
@@ -4,19 +4,19 @@
       foo <- new_class("foo")
       foo(1)
     Error <simpleError>
-      All arguments to <foo> constructor must be named
+      unused argument (1)
```

Comment:
Are ok with these error messages now being less informative?

I think so since the constructor now has the properties as full arguments, but wanted to double check

Reply:
Yeah, it's a bit of a bummer (especially since I just wrote this code 😆), but I think it's a net win.